esp/src/eclwatch/XrefFoundFilesWidget.js

```diff
@@ -121,7 +121,7 @@ define([
             var selections = this.grid.getSelected();
             var list = this.arrayToList(selections, "Name");
             if (confirm(this.i18n.AddTheseFilesToDali + "\n" + list)) {
-                WsDFUXref.DFUXRefArrayAction(selections, this.i18n.Attach, context.params.Name, "Attach").then(function (response) {
+                WsDFUXref.DFUXRefArrayAction(selections, this.i18n.Attach, context.params.Name, "Found").then(function (response) {
```

Comment:
this.i18n.Attach should just have "Attach". Note** not an issue you introduced but there is where the action goes in and type is at the end.